open-cluster-management/backlog#9648 clusterrole21

cadawson opened the pull request.

cadawson committed:
open-cluster-management/backlog#9648 clusterrole21

cadawson commented:
Hi @skeeey. Here is the 2.1 version of the clusterroles. Can you take a look? Thank you!

skeeey commented on manage_cluster/define_clusterrole.adoc:
```diff
@@ -0,0 +1,79 @@
+  - register.open-cluster-management.io
+  resources:
+  - managedclusters/accept
+  resourceNames:
```
We should remove `resouceNames` field in 2.1

cadawson committed:
open-cluster-management/backlog#9648 rev cmts

cadawson commented:
Hi @skeeey . Thank you. I think I removed the wrong one. I removed the last one and added the one that I removed earlier. I believe you are saying that only the one instance after managedclusters/accept should be removed. Let me know if all of them should be removed. There are 2 other instances of resourceNames in the content. Thank you.

skeeey commented:
@cadawson yeah, it is right this time
